internal/operands/metrics/reconcile.go

```diff
@@ -59,6 +67,47 @@ const (
 	operandComponent = common.AppComponentMonitoring
 )
 
+func reconcilePrometheusMonitor(request *common.Request) (common.ReconcileResult, error) {
+	return common.CreateOrUpdate(request).
+		ClusterResource(newServiceMonitorCR(request.Namespace)).
+		WithAppLabels(operandName, operandComponent).
+		UpdateFunc(func(newRes, foundRes client.Object) {
+			foundRes.(*promv1.ServiceMonitor).Spec = newRes.(*promv1.ServiceMonitor).Spec
+		}).
+		Reconcile()
+}
+
+func reconcileMonitoringRbacRole(request *common.Request) (common.ReconcileResult, error) {
+	return common.CreateOrUpdate(request).
+		ClusterResource(newMonitoringRbacRole(request.Namespace)).
+		WithAppLabels(operandName, operandComponent).
+		UpdateFunc(func(newRes, foundRes client.Object) {
+			foundRes.(*rbac.Role).Rules = newRes.(*rbac.Role).Rules
+		}).
+		Reconcile()
+}
+
+func reconcileMonitoringRbacRoleBinding(request *common.Request) (common.ReconcileResult, error) {
+	return common.CreateOrUpdate(request).
+		ClusterResource(newMonitoringRbacRoleBinding(request.Namespace)).
```

Comment:
Similarly, can this be `NamespacedResource()`?

Reply:
done.